src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -155,11 +159,24 @@ export class RelayFeeCalculator {
 
     return defaultFee;
   }
-  async relayerFeeDetails(amountToRelay: BigNumberish, tokenSymbol: string, tokenPrice?: number) {
+  async relayerFeeDetails(
+    amountToRelay: BigNumberish,
+    tokenSymbol: string,
+    destinationChainId: number,
+    tokenPrice?: number
+  ) {
     let isAmountTooLow = false;
-    const gasFeePercent = await this.gasFeePercent(amountToRelay, tokenSymbol, tokenPrice);
+    const rawGasFeePercent = await this.gasFeePercent(amountToRelay, tokenSymbol, tokenPrice);
+    const gasFeeDiscountPercent = this.gasDiscountPercent[destinationChainId]
+      ? toBN(100).sub(this.gasDiscountPercent[destinationChainId])
+      : toBN(100);
```

Comment:
Is there a naming clash here? We seem to specify gasDiscountPercent during the relayerFeeCalculator() constructor, but here we sub that amount from 100. Wouldn't that instead make it a multiplier, rather than a discount?